simulator/src/main/java/com/hazelcast/simulator/testcontainer/PrimordialRunStrategy.java

```diff
@@ -13,7 +13,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.hazelcast.simulator.worker;
+package com.hazelcast.simulator.testcontainer;
+
+import com.hazelcast.simulator.worker.RunStrategy;
```

Comment:
`PrimordialRunStrategy` is in `testcontainer`, but `RunStrategy` is in `worker`?